pkgs/development/python-modules/mbzero/default.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  requests,
+  requests-oauthlib,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "mbzero";
+  version = "0.4";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-4ZlX2dTTXF3Zc5vUymeS+3vXd7+IBnA4tNsvVKNj+ZI=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    requests
+    requests-oauthlib
+  ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
```

Comment:
can you add `pythonCheckImports`?

Reply:
sorry i missed this

while i couldn't find `pythonCheckImports` in the manual i added `pythonImportsCheckHook` which is probably what you meant